app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+from __future__ import division
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(
+                km=self.km + other.km
+            )
+        else:
+            return Distance(
+                km=self.km + other
+            )
+
+    def __iadd__(self, other: Distance) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+
+        return self
+
+    def __mul__(self, other: float) -> Distance:
+        return Distance(
+            km=self.km * other
+        )
+
+    def __truediv__(self, other: float) -> Distance:
+        return Distance(
+            km=round(self.km / other, 2)
+        )
+
+    def __lt__(self, other: float) -> bool:
```

Comment:
you didn't understand me. I meant that you can avoid only `else`, because return ends function.
The correct variant will be:

```
   def __gt__(self, other: Distance | int | float) -> bool:
        if isinstance(other, Distance):
            return self.km > other.km
        return self.km > other
```

Reply:
don't forget avout annotations